apps/browser-extension-wallet/src/providers/AnalyticsProvider/analyticsTracker/AnalyticsTracker.ts

```diff
@@ -44,19 +53,23 @@ export class AnalyticsTracker {
   }
 
   async sendPageNavigationEvent(): Promise<void> {
+    if (POSTHOG_OPTED_OUT_EVENTS_DISABLED && !this.isOptedInUser) return;
```

Comment:
1. Could we use UserIdService changes from #375 to avoid introducing additional property?
```suggestion
    if (POSTHOG_OPTED_OUT_EVENTS_DISABLED && this.userIdService.getTrackingType() === UserTrackingType.Basic) return;
```
2. Could we extract this line to a helper method to reduce duplication?